### ISSUE TITLE
Hat board frequency 

### ISSUE THREAD

**Issue by andreltr** [sbcshop/Lora-HAT-for-Raspberry-Pi#2]
Hello,

Is it possible to change a board frequency from the code provided?  For instance, to the 915MHz range?

Thanks,

Andre Luiz

**Comment by sbcshop1** [comment]
No, we can not change the Lora Hat frequency, we provide three frequencies Lora HAT  board 433, 868 , and  915MHz
